packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -101,29 +100,28 @@ export function autocomplete<TItem extends BaseItem>(
 
   function runRender() {
     const renderProps = {
-      isTouch: isTouch.value,
-      state: lastStateRef.current,
       autocomplete: autocomplete.value,
-      propGetters,
-      dom: dom.value,
+      autocompleteScopeApi,
       classNames: props.value.renderer.classNames,
+      container: props.value.renderer.container,
+      dom: dom.value,
+      isTouch: isTouch.value,
       panelContainer: isTouch.value
         ? dom.value.touchOverlay
         : props.value.renderer.panelContainer,
-      autocompleteScopeApi,
+      pragma: props.value.renderer.pragma,
+      pragmaFrag: props.value.renderer.pragmaFrag,
+      propGetters,
+      state: lastStateRef.current,
     };
 
     renderSearchBox(renderProps);
     renderPanel(props.value.renderer.render, renderProps);
   }
 
   function scheduleRender(state: AutocompleteState<TItem>) {
-    if (renderRequestIdRef.current !== null) {
-      cancelAnimationFrame(renderRequestIdRef.current);
-    }
-
     lastStateRef.current = state;
-    renderRequestIdRef.current = requestAnimationFrame(runRender);
+    runRender();
```

Comment:
We don't need to take care of `requestAnimationFrame` ourselves now because Preact embeds this logic.